### PR TITLE
Cleanup: Brackets with double quotes to include structures ( like wirem…

### PR DESCRIPTION
**Pull request by dvdvideo1234**
…od )

Converted: Global `constraints` and `serializable` to real globals

### CODE REVIEW

**Comment by bjcscat** [comment]
whats the reasoning here?

**Review by thegrb93 — APPROVED** [review]

**Comment by dvdvideo1234** [comment]
> whats the reasoning here?

Pretty simple actually.

In wiremod there is a convention for all `include` directives to use brackets and double quotes. This will not harm the mod in any way. For relocating the `constraints` and `serializable` it will be easier for future support. When I searched for `constraints` I literally got a bunch of results and the variable was used in only one place. Either way all control compare values are stored in `gtSetupTable`, so why not the other two ;)

File: lua/advdupe2/sv_clipboard.lua [hunk 1]
```diff
@@ -15,27 +15,63 @@ AdvDupe2.JobManager.Queue = {}
...
+		["angle"   ] = true,
+		["Ang"     ] = true,
+		["Angle"   ] = true
+	},
+	MODEL = {
+		["model"   ] = true,
+		["Model"   ] = true
+	},
+	PLAYER = {
+		["pl"      ] = true,
```
Comment (wrefgtzweve) [inline lua/advdupe2/sv_clipboard.lua:57]: it'd probably look better if you added the extra spaces between the ] and =

Comment (dvdvideo1234) [reply lua/advdupe2/sv_clipboard.lua]: Fixed in https://github.com/wiremod/advdupe2/pull/445/commits/f890649614b9a38a0b5fe48d3d79440d18936e5e

**Pull request merged**